docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "2"
+
+services:
+  xo:
+    build:
+      context: ./
+    volumes:
+      - .:/go/src/github.com/knq/xo
+  postgres:
+    image: postgres:9.6
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_USER: xodb
+      POSTGRES_PASSWORD: xodb
+  mysql:
+    image: mysql:5.6
+    ports:
+      - 3306:3306
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: "true"
+      MYSQL_DATABASE: xodb
+      MYSQL_USER: xodb
+      MYSQL_PASSWORD: xodb
+  mssql:
+    image: microsoft/mssql-server-linux
+    ports:
+      - 1433:1433
+    volumes:
+      - ./volumes/mssql/:/var/opt/mssql/
+    environment:
+      ACCEPT_EULA: "Y"
+      SA_PASSWORD: P@55w0rd
+  oracle:
+    image: sath89/oracle-12c
```

Comment:
what's different in this image vs the base image you used for the docker file?

Reply:
`sath89/oracle-12c` is an image which includes oracle database server.
We need oracle client libraries to access the database in order to generate models. Sine it's difficult to download oracle client library by curl, so I use an image which has built-in client. So the difference is client and server.